src/test/java/eu/xenit/gradle/docker/alfresco/internal/amp/ModuleInformationAmpTest.java

```diff
@@ -0,0 +1,45 @@
+package eu.xenit.gradle.docker.alfresco.internal.amp;
+
+import static org.junit.Assert.*;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+public class ModuleInformationAmpTest {
+
+    @Rule
+    public final TemporaryFolder temporaryFolder = new TemporaryFolder();
+
+    @Test
+    public void createFromAmp() throws IOException {
+        File ampFolder = temporaryFolder.newFolder("test-amp");
+        Path ampFile = ampFolder.toPath().resolve("test123.amp");
+        Files.copy(getClass().getResourceAsStream("/test123.amp"), ampFile);
```

Comment:
creating an amp from code (for example a static method) would be more transparent - it's part of the test-setup I can't see now

Reply:
Good point. I will see what I can do, but creating zips in java is a bit more work than manually crafting one :smile: